eland/ml/exporters/es_gb_models.py

```diff
@@ -0,0 +1,449 @@
+#  Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Is this module supported to be public, or can we make it private and expose the classes via `eland.ml.ml_model`?

Reply:
I am not sure what you mean here. Users can use the ESGradientBoosting* objects directly, accessing them from `pipeline['es_model']`. Can you elaborate?